src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started to work");

```

Comment:
The message 'Bulldozer started to work' should be more informative. It should indicate what type of machine is working. Consider including the action it performs, for example, 'Bulldozer is moving earth'.